npk/src/npk.rs

```diff
@@ -302,7 +313,7 @@ impl<R: io::Read + io::Seek> Npk<R> {
         }
     }
 
-    pub fn into_inner(self) -> ZipArchive<R> {
+    pub async fn into_inner(self) -> ZipArchive<std::fs::File> {
```

Comment:
Shouldn't this return a `tokio::fs::File` since that's use for the construction of a `Zip`?